client/css/style.css

```diff
@@ -1614,8 +1624,8 @@ kbd {
 	animation-fill-mode: forwards;
 	-webkit-animation-timing-function: ease-in;
 	animation-timing-function: ease-in;
-	-webkit-animation-delay: .4s;
-	animation-delay: .4s;
+	-webkit-animation-delay: .1s;
+	animation-delay: .1s;
```

Comment:
Yes, I read your PR description 😅

However, your changes are not on an animation _duration_ but on an animation _delay_. There is a reason we have a 0.4s delay on tooltips, it's to avoid the flickering when moving the mouse over by chance.

We need to keep this at `.4s` and I won't be able to 👍 until they're reverted. Would you mind doing so? 😃